utility/src/main/java/org/oppia/android/util/parser/svg/BUILD.bazel

```diff
@@ -0,0 +1,94 @@
+"""
+General-purpose utilities for vector drawables and transcoders.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "block_picture_drawable",
+    srcs = [
+        "BlockPictureDrawable.kt",
+    ],
+    visibility = ["//utility/src/main/java/org/oppia/android/util/parser/image:__pkg__"],
```

Comment:
Glad to see this is working. What about for the other libraries in this package? I suspect they can also have restricted visibility.

Reply:
privatized all the imports